### PR TITLE
Getstarted Oak Environment

### PR DESCRIPTION
**Pull request by giobart**

### CODE REVIEW

File: content/docs/getting-started/oak-environment/your-first-orchestrator.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+If the startup succeeds, **congratulations! 🎉🎉**
+You're now ready to add your first worker node to your cluster (see: [Add Edge Devices (Workers) to Your Setup](/docs/getting-started/oak-environment/add-edge-devices-workers-to-your-setup/)).
+
+### Shutdown the Components
+
+To stop your **Root & Cluster** orchestrator components run:
+
+```bash
+docker compose -f ~/oakestra/compose-single-machine.yaml down
```
Comment (Mjaethers) [inline content/docs/getting-started/oak-environment/your-first-orchestrator.md]:
```suggestion
docker compose -f ~/oakestra/1-DOC.yaml down
```
I have to run this command to shut it down again

File: content/docs/getting-started/oak-environment/your-first-worker.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+```bash
+sudo NodeEngine -a <IP address of your cluster orchestrator> -d
+```
+
+{{< callout context="note" title="Note" icon="outline/info-circle" >}}
+the `-d` flag runs the NodeEngine in background (detached mode)
+{{< /callout >}}
+
+Check if your worker is running, and it's correctly registered to your cluster:
```
Comment (Mjaethers) [inline content/docs/getting-started/oak-environment/your-first-worker.md]:
```suggestion
Check if your worker is running, and that it's correctly registered to your cluster:
```

File: content/docs/getting-started/oak-environment/your-first-worker.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+```bash
+sudo NodeEngine status
+```
+
+If everything is showing up green 🟢... Congratulations, your worker node is set up! 🎉
+
+[SCREENSHOT HERE]
```
Comment (Mjaethers) [inline content/docs/getting-started/oak-environment/your-first-worker.md]: I suspect you forgot a todo here

File: content/docs/getting-started/oak-environment/arch-new.png
Comment (Mjaethers) [inline content/docs/getting-started/oak-environment/arch-new.png:1]: image
In dark mode the white boxes behind the black text looks a little weird (but does make it legible!), maybe make the text grey instead?

**Review by Mjaethers — COMMENTED** [review]
Look really nice! I only have some minor suggestions

File: content/docs/getting-started/oak-environment/your-first-orchestrator.md (hunk 1 above)
Comment (giobart) [reply content/docs/getting-started/oak-environment/your-first-orchestrator.md:49]: where is this `compose-single-machine.yaml` coming from?

File: content/docs/getting-started/oak-environment/your-first-worker.md (hunk 3 above)
Comment (giobart) [reply content/docs/getting-started/oak-environment/your-first-worker.md]: thanks for noticing! Fixed

File: content/docs/getting-started/oak-environment/arch-new.png
Comment (giobart) [reply content/docs/getting-started/oak-environment/arch-new.png:1]: done, grey works well! ty

File: content/docs/getting-started/oak-environment/running.png
Comment (nitindermohan) [inline content/docs/getting-started/oak-environment/running.png:1]: Is this the screenshot from oak-cli? Please remove any identifying information from here (e.g. public IP, hostname, etc,)

File: content/docs/getting-started/oak-environment/your-first-orchestrator.md
Comment (nitindermohan) [inline content/docs/getting-started/oak-environment/your-first-orchestrator.md:1]: Is the get-started script installing oak-cli now? If yes, also point to README for oak-cli

**Review by nitindermohan — COMMENTED** [review]
Overall really looks good but some minor clarifications required

File: content/docs/getting-started/oak-environment/your-first-orchestrator.md
Comment (nitindermohan) [reply content/docs/getting-started/oak-environment/your-first-orchestrator.md:1]: ignored

File: content/docs/getting-started/oak-environment/running.png
Comment (giobart) [reply content/docs/getting-started/oak-environment/running.png:1]: I'll remove cmhsotname and the IP address

**Comment by giobart** [comment]
All suggestions have been integrated.

**Review by Mjaethers — APPROVED** [review]

**Review by nitindermohan — APPROVED** [review]
Looks good. Approved 🎉

**Pull request merged**